server/index.js

```diff
@@ -1,10 +1,58 @@
+const path = require('path');
 const express = require('express');
+const query = require('./database/queries');
 
 const app = express();
 const port = 3000;
 
-app.get('/', (req, res) => {
-  res.send('Hello World!');
+app.get('/api/house/:id', (req, res) => {
```

Comment:
Your API endpoints look well thought out to me. I will probably use yours for inspiration on mine as well.